research_town/agents/agent.py

```diff
@@ -297,3 +399,38 @@ def write_rebuttal(
             q5=q5_result.get('q5', ''),
         )
         return rebuttal
+
+    def write_rebuttal_paper_text(
```

Comment:
maybe unify the api to be write_rebuttal_from_proposal and write_rebuttal_from_paper. write_revie_from_proposal and write_review_from_paper.